ui/components/ui/box/box.stories.js

```diff
@@ -582,8 +582,8 @@ export const ResponsiveProps = () => {
         marginBottom={[0]}
         padding={[2, 4]}
         gap={[2, 4]}
-        display={['flex', null, null, 'none']}
-        flexDirection={['column', 'row']}
+        display={[DISPLAY.FLEX, null, null, DISPLAY.NONE]}
+        flexDirection={['column', 'column', 'row']}
```

Comment:
Oh could do the same here :) 

```suggestion
        flexDirection={[FLEX_DIRECTION.COLUMN, FLEX_DIRECTION.COLUMN, FLEX_DIRECTION.ROW]}
```